src/pages/MerchandiseList/Card.tsx

```diff
@@ -36,12 +38,26 @@ const Card = ({ productId, imgSrc, text, price, sizeRange }: CardProps) => {
               <Text noOfLines={2}>{text}</Text>
               <Text align="center">{displayPrice(price)}</Text>
             </Flex>
+            {isAvailable && !isOutOfStock && (
             <Flex textColor="gray.400" justifyContent="space-between" mt={1} alignItems="center">
               <Text fontWeight={600} textTransform="uppercase" fontSize={["xs", "sm"]}>
                 {sizeRange}
               </Text>
               <ArrowForwardIcon />
             </Flex>
+            )}
+            <Flex justifyContent="left" mt={1} alignItems="center">
+                {!isAvailable && (
+                  <Badge color="grey" mr={2} fontSize="xs" variant="outline" display="inline">
+                    unavailable
+                  </Badge>
+                )}
+                {isOutOfStock && (
+                  <Badge color="grey" mr={2} variant="outline" display="inline">
+                    out of stock
+                  </Badge>
+                )}
+            </Flex>
```

Comment:
not sure if having both `unavailable` and `out of stock` badges looks crowded, we can consider prioritising either badge over the other and just displaying one.